### ISSUE TITLE
[Fixed in GH-6052] Action buttons being affected by theme Calydon fix in the cxbase

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#5535]
Recently a `cxbase` fix was introduced in theme Calydon to address lost visual focus in buttons, this fix is causing unexpected result on *Action Buttons* (as shown in screenshots below) once activating the theme.

![Screen Shot 2019-11-22 at 11 53 22 AM](https://user-images.githubusercontent.com/15113219/69649987-e50a6500-103b-11ea-83c6-95869c2344aa.png)

![Screen Shot 2019-11-26 at 11 26 32 AM](https://user-images.githubusercontent.com/15113219/69652969-75e33f80-1040-11ea-9553-a714a0494b93.png)

**Comment by developpeurweb** [comment]
Deleting this line should do it but first find out why it was included. Primary and Secondary buttons seemed to be the target of previous fix.

![Screen Shot 2019-11-26 at 11 57 28 AM](https://user-images.githubusercontent.com/15113219/69655176-01aa9b00-1044-11ea-8e2e-91689314326c.png)

**Comment by bgambocjaviniar** [comment]
<img width="1438" alt="Screen Shot 2019-12-02 at 4 38 18 PM" src="https://user-images.githubusercontent.com/29099466/69999897-c3234d80-1527-11ea-9351-fb7ee539e42f.png">

**Comment by developpeurweb** [comment]
Seems all good now
- Theme `calydon` should be enabled to see changes.

![Screen Shot 2020-01-30 at 2 48 19 PM](https://user-images.githubusercontent.com/15113219/73484771-e758aa80-436f-11ea-8e49-497ded5e91be.png)
![Screen Shot 2020-01-30 at 2 49 09 PM](https://user-images.githubusercontent.com/15113219/73484772-e758aa80-436f-11ea-97e8-dd510d89720f.png)
![Screen Shot 2020-01-30 at 2 50 04 PM](https://user-images.githubusercontent.com/15113219/73484774-e7f14100-436f-11ea-9a08-9b22016e3122.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
